cmd/calculator.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/coreos/kube-aws/cluster"
+	"github.com/coreos/kube-aws/config"
+	"github.com/spf13/cobra"
+)
+
+//TODO this is a first step to calculate the stack cost
+//this command could scrap aws to print the total cost, rather just showing the link
+var (
+	cmdCalculator = &cobra.Command{
+		Use:          "calculator",
+		Short:        "Discovery the monthly cost of your cluster",
+		Long:         ``,
+		RunE:         runCmdCalculator,
+		SilenceUsage: true,
+	}
+
+	calculatorOpts = struct {
+		awsDebug bool
+	}{}
+)
+
+func init() {
+	RootCmd.AddCommand(cmdCalculator)
+	cmdCalculator.Flags().BoolVar(&calculatorOpts.awsDebug, "aws-debug", false, "Log debug information from aws-sdk-go library")
+}
+
+func runCmdCalculator(cmd *cobra.Command, args []string) error {
+
+	conf, err := config.ClusterFromFile(configPath)
+	if err != nil {
+		return fmt.Errorf("Failed to read cluster config: %v", err)
+	}
+
+	opts := stackTemplateOptions("", false)
+
+	cluster, err := cluster.NewCluster(conf, opts, calculatorOpts.awsDebug)
+	if err != nil {
+		return fmt.Errorf("Failed to initialize cluster driver : %v", err)
+	}
+
+	if err := cluster.ValidateUserData(); err != nil {
+		return err
+	}
+
+	out, err := cluster.Calculator()
```

Comment:
nit: I'd rather like to name the func `EstimateCost()` instead of `Calculator()` so that we can make sure that it doesn't return a calculator to do a calculation but return the result of an estimation(=estimated cost)!